script/main.js

```diff
@@ -16,3 +68,111 @@ menuItems.forEach((item) => {
+      modalFeaturedImage.src = work.featuredImage;
+
+      const modalCloseIcon = addElement('a', modalWorkCard, 'close-icon');
+      modalCloseIcon.classList.add('fa');
+      modalCloseIcon.classList.add('fa-times');
+
+      if (isDisplayOrMobile) {
+        const modalWorkCardDisplay = addElement('div', modalWorkCard, 'work-card-display');
+        const modalWorkCardTitle = addElement('h3', modalWorkCardDisplay, 'work-card-title-display');
+        modalWorkCardTitle.innerHTML = work.name;
+
+        const modalWorkCardButton1 = addElement('a', modalWorkCardDisplay, 'modal-work-card-button-display');
+        modalWorkCardButton1.innerHTML = 'See Live';
+        modalWorkCardButton1.href = work.linkToLiveVersion;
+
+        const modalWorkCardButtonImg1 = addElement('img', modalWorkCardButton1, 'modal-work-card-button-img');
+        modalWorkCardButtonImg1.src = './img/see live icon.png';
+
+        const modalWorkCardButton2 = addElement('a', modalWorkCardDisplay, 'modal-work-card-button-display');
+        modalWorkCardButton2.innerHTML = 'See Source';
+        modalWorkCardButton2.href = work.linkToSource;
+
+        const modalWorkCardButtonImg2 = addElement('img', modalWorkCardButton2, 'modal-work-card-button-img');
+        modalWorkCardButtonImg2.src = './img/see source icon.png';
+      } else {
+        const modalWorkCardTitle = addElement('h3', modalWorkCard, 'work-card-title');
+        modalWorkCardTitle.innerHTML = work.name;
+      }
+
+      const modalWorkCardTags = addElement('ul', modalWorkCard, 'work-card-tags');
+      modalWorkCardTags.classList.add('nobull');
+
+      workTechnologies.forEach((technology) => {
+        const modalWorkCardTag = addElement('li', modalWorkCardTags, 'work-card-tag');
+        modalWorkCardTag.innerHTML = technology;
+      });
+
+      const modalWorkCardDescr = addElement('p', modalWorkCard, 'work-card-descr');
+      modalWorkCardDescr.innerHTML = work.description;
+
+      if (!isDisplayOrMobile) {
+        const modalWorkCardButton1 = addElement('a', modalWorkCard, 'modal-work-card-button');
+        modalWorkCardButton1.innerHTML = 'See Live';
+        modalWorkCardButton1.href = work.linkToLiveVersion;
+
+        const modalWorkCardButtonImg1 = addElement('img', modalWorkCardButton1, 'modal-work-card-button-img');
+        modalWorkCardButtonImg1.src = './img/see live icon.png';
+
+        const modalWorkCardButton2 = addElement('a', modalWorkCard, 'modal-work-card-button');
+        modalWorkCardButton2.innerHTML = 'See Source';
+        modalWorkCardButton2.href = work.linkToSource;
+
+        const modalWorkCardButtonImg2 = addElement('img', modalWorkCardButton2, 'modal-work-card-button-img');
+        modalWorkCardButtonImg2.src = './img/see source icon.png';
+      }
+
+      modalCloseIcon.addEventListener('click', () => {
+        modalWorkContainer.classList.add('hide');
+        modalWorkContainer.removeChild(modalWorkContainer.firstElementChild);
```

Comment:
### Your Desktop Modal Close Button
![desktop-close-button](https://user-images.githubusercontent.com/93039359/196652587-5d306d2d-dc1e-4b79-87f5-70a8ba6935ef.PNG)

### Figma Desktop Modal Close Button
![Figma-Desktop-Design](https://user-images.githubusercontent.com/93039359/196652615-2d6c9cdc-96f3-4fdd-8337-bc5d2c13bdd7.PNG)

- [ ] Your close button is not very similar to the Figma template in desktop version. Kindly make it look similar to the Figma template to make your design professional 😊

Reply:
Thank you, it's done.